project.clj

```diff
@@ -11,14 +11,15 @@
                  [byte-streams "0.2.2"]
                  [org.clojure/math.combinatorics "0.1.3"]
                  [medley "0.8.3"]]
+  :plugins [[lein-cljfmt "0.5.7"]]
```

Comment:
This adds the plugin in two places, which you probably shouldn't do. If you must add it in one (because e.g. you don't want to type `lein with-profile +test ...`, like Travis does), you should add it to `:dev`, a profile which is enabled by default.

Reply:
I recommend that you start by removing this line, because that will make it a lot easier to resolve conflicts (just merge master forward).

Reply:
agreed. I am still new to clojure and lein. so I didn't really know how to initiate cljfmt. I looked it up online and that's the best I could come up with. I will remove that line.